tools/arch/PKGBUILD

```diff
@@ -39,7 +39,8 @@ build() {
     --disable-dependency-tracking \
     --disable-silent-rules \
     --with-cockpit-user=cockpit-ws \
-    --with-cockpit-ws-instance-user=cockpit-wsinstance
+    --with-cockpit-ws-instance-user=cockpit-wsinstance \
+    --with-default-session-path=/usr/local/sbin:/usr/local/bin:/usr/bin
```

Comment:
This is a bit yucky -- we'd need to make sure that this lands in the official arch packaging, and also it will break `make install` on Arch. Could configure be a bit more clever about this, and not include /sbin and /usr/sbin if these are symlinks? There might still be situations where this does the wrong thing, but these are much more corner-case-y.

Reply:
Reasoning: /sbin being a symlink to /usr/sbin or to /bin or to /usr/bin/ is usually a distro-wide decision. If admins do weird things on their system and mess around in /usr and /bin, they get to keep both halves. And if /sbin is a symlink to another dir which is already in PATH, there is no point including it into $PATH in addition, it just makes things slower.

Reply:
Yes, good point!

Reply:
Done, in a quite Arch specific way.